lib/classes/Utils.js

```diff
@@ -197,6 +200,25 @@ class Utils {
                   count: 1,
                 });
               }
+
+              // For HTTP events, see what authorizer types are enabled
+              if (event.http && event.http.authorizer) {
```

Comment:
I know restructuring does not add a huge amount of performance, but within a loop and with so many deep object key/value pairs being accessed, I think it might

1) help this code a little more concise within the multilpe if checks
2) perhaps add very minimal performance increase